backend/db/initialize.py

```diff
@@ -22,6 +22,35 @@
 
 SCHEMA = CONFIG['schema']
 
+DDL_MANAGE = """CREATE TABLE IF NOT EXISTS public.manage (
+key text not null, 
+value text);"""
+DDL_COUNTS = """CREATE TABLE IF NOT EXISTS public.counts (
+timestamp text not null, 
+date text, 
+schema text not null, 
+"table" text not null, 
+count integer not null, 
+delta integer not null);"""
+DDL_COUNTS_RUNS = """CREATE TABLE IF NOT EXISTS public.counts_runs (
+timestamp text not null, 
+date text, 
+schema text not null, 
+note text);"""
+# fetch_audit: table schema
+#   table: text; examples: code_sets | concept_set_container | concept_set_members | concept_set_version_item
+#   primary_key: text; comma-delimited: this should store the value of the keys, e.g. codeset_id 12345, concept_id 5678
+#     would be 12345,5678
+#   status_initially: text; factor: success | fail-excessive-members | fail-0-members | fail-unknown?
+#   success_datetime: timestamp; initially null, used by code that updates the derived tables.
+#   comment: text
+# todo: change comment to text[]?
+DDL_FETCH_AUDIT = """CREATE TABLE IF NOT EXISTS public.fetch_audit (
```

Comment:
@Sigfried I set up `initialize()` to create the `fech_audit` table, renamed from plain 'audit' for [disambiguation](https://dba.stackexchange.com/a/15187) purposes.

Some notes about the columns:
- `table` (text)
- `primary_key` (text): We initially wrote down to create two of these columns; one for the names of the primary keys in their tables, and one for the PK value(s) for a particular record. But we don't need the former, since `enclave_wrangler/[config.py](http://config.py/)` lists them now. So now it just shows values, comma-delimited, e.g. codeset_id 12345, concept_id 5678 would be: 12345,5678.
- `status_initially` (text): Renamed from 'status'. This is to disambiguate from the initial status when we first fetch, from the final status, which we can ascertain from 'success_datetime'. Right now I think it will be one of:  success | fail-excessive-members | fail-0-members | fail-unknown.
- `success_datetime` (datetime): Renamed from 'processed'. Nullable.
- `comment` (text): Left as plain text for now but can change to array if needed.

_Didn't need this column:_
- `fixed_later` (bool): Don't need this column because we can ascertain by if 'success_datetime' is not null

Reply:
I'm going to continue using this table schema, but upon actually trying to put it into practice, I realize that it suits more the use case you had in mind than the one that I'm immediately working on. Rather than 'table' and 'primary key', what I need to know specifically is what fetched link object failed (e.g. expression items, members) for a given code set.